src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/eltwise.cpp

```diff
@@ -750,6 +750,51 @@ const auto params_4D_bitwise_NOT_i32 = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_Bitwise_NOT_i32, EltwiseLayerCPUTest, params_4D_bitwise_NOT_i32, EltwiseLayerCPUTest::getTestCaseName);
 
+const auto params_4D_bitwise_shift = ::testing::Combine(
```

Comment:
Please move these test cases to src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common/eltwise.cpp, due to the current implementation should work on all platforms.

Reply:
Moved to the common (src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common/bitwise_shift.cpp)